phpunit/functional/LockedfieldTest.php

```diff
@@ -924,4 +924,48 @@ public function testReplaceLockedField()
         $this->assertTrue($computer->getFromDB($cid));
         $this->assertEquals($computer->fields['locations_id'], 0);
     }
+
+    public function testCheckAllInventoryLockableObjects()
+    {
+        $this->login('glpi', 'glpi');
+
+        /** @var \DBmysql $DB */
+        global $DB;
+
+        /** @var array $CFG_GLPI */
+        global $CFG_GLPI;
+        foreach ($CFG_GLPI['inventory_lockable_objects'] as $itemtype) {
+            $this->assertTrue($DB->fieldExists($itemtype::getTable(), 'is_dynamic'));
```

Comment:
```suggestion
            $this->assertTrue($DB->fieldExists($itemtype::getTable(), 'is_dynamic'), "Add a clear message here in case of validation failure.");
```

can you add message here